1.50.0/windowsservercore-1809/gnu/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:1809
+
+SHELL ["powershell.exe", "-NoLogo", "-ExecutionPolicy", "Bypass", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
It looks like golang's image doesn't set `-NoLogo` or `-ExecutionPolicy Bypass`: https://github.com/docker-library/golang/blob/master/1.15/windows/windowsservercore-1809/Dockerfile#L10. Do you know why there's a difference there?

Reply:
I set `-NoLogo` because I don't see a reason to have the PowerShell version information and copyright info printed at the start of each command as it's just noise.

As for the `-ExecutionPolicy Bypass`, I think that might be a leftover from another Windows docker tutorial I had followed. I don't see a reason it needs to be kept.